kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -126,28 +127,28 @@
       'icon-button': require('kolibri.coreVue.components.iconButton'),
     },
     props: [
-      'userid', 'username', 'fullname', 'roles', // TODO - validation
+      'user',
```

Comment:
I don't think it's a good idea to collapse all the explicit input properties into a single input 'user' object, because it makes the contract less explicit, and it also couples the implementation to the current structure of the vuex state. Let's break it out again, giving each required input property proper type validation.